### ISSUE TITLE
customMetadataTypeAccesses permission disappears when executing sfdx dxb:profile:convert

### ISSUE THREAD

**Issue by HingChow** [davidbrowaeys/DXB#5]
### Steps to reproduce:
1. Create a custom metadata type in a scratch org.
2. Assign permissions to the custom metadata type to the profile.
3. Pull the profile changes into your source code.
3. Run `sfdx dxb:profile:convert`.

### Expected Result:
A JSON file containing the custom metadata type is created.

### Actual Result:
No JSON file containing the custom metadata type is created.

Note: An example of the profile permission are as follows:
```
...
<customMetadataTypeAccesses>
    <enabled>true</enabled>
    <name>Some_Custom_Metadata__mdt</name>
</customMetadataTypeAccesses>
...
```

**Comment by davidbrowaeys** [comment]
Hi @HingChow , made some fixes in latest version 1.5.3. I also move all the user permission under the main profile to reduce the number of files generated. Let me know how you go. Interested to see how people are using this.

**Comment by HingChow** [comment]
It works! Thanks for your help. 

We use your utility to trim profile permissions who don't want to move to higher environments. Unfortunately, our scratch orgs have a different Salesforce edition than higher environments which causes deployment issues.
